pkg/kritis/metadata/containeranalysis/containeranalysis_int_test.go

```diff
@@ -137,3 +140,30 @@ func TestCreateAttestationNoteAndOccurrence(t *testing.T) {
 		}
 	}
 }
+
+func TestGetMultiplePages_Vulnerabilities(t *testing.T) {
+	d, err := New()
+	if err != nil {
+		t.Fatalf("Could not initialize the client %s", err)
+	}
+
+	// Set PageSize to 300
+	createListOccurrencesRequest = createListOccurrencesRequest_test
+
+	vuln, err := d.Vulnerabilities("gcr.io/kritis-int-test/java-with-vulnz@sha256:358687cfd3ec8e1dfeb2bf51b5110e4e16f6df71f64fba01986f720b2fcba68a")
```

Comment:
How many vulz counts is this image? can we make sure to have 3+ pages?

Reply:
According to http://gcr.io/kritis-int-test/java-with-vulnz@sha256:358687cfd3ec8e1dfeb2bf51b5110e4e16f6df71f64fba01986f720b2fcba68a, there are 931 vulnerabilities as of now. I checked and the length of the vuln array is 931